finagle-http/src/main/scala/com/twitter/finagle/http/Status.scala

```diff
@@ -29,7 +29,47 @@ case class Status(code: Int) {
     }
 }
 
+
+
+
+
 object Status {
+
+  @inline
+  private[finagle] def inRange(lower: Int, upper: Int)
+                              (status: Status): Option[Status] =
```

Comment:
No, not really. It was like that because I wanted to write the unapply function like this:
```scala
def unapply: Status => Option[Status] =
  inRange(100,200)
```
But it appears that leads to several compile errors.